x-pack/plugins/cases/common/api/cases/status.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import * as rt from 'io-ts';
-import { CaseStatuses } from '@kbn/cases-components';
+import { CaseStatuses } from '@kbn/cases-components/src/status/types';
```

Comment:
Wasn't aware that not importing from exact folder increases bundle size 😄

Reply:
Ideally, it wouldn't because web pack should be able to recognize what is and is not necessary. But it seems we can't fully trust it 😬

Reply:
Haha.. better to be safe!! 👍